toolkit/tools/scheduler/schedulerutils/implicitprovides.go

```diff
@@ -111,7 +111,7 @@ func implicitPackagesToUnresolvedNodesInGraph(pkgGraph *pkggraph.PkgGraph, useCa
 		// Only consider these as resolved if useCachedImplicit is set.
 		if n.State == pkggraph.StateCached {
 			if useCachedImplicit {
-				continue
+				logger.Log.Warnf("Implicit node (%s) was already cached and may have been used to satisfy another node (useCachedImplicit=true). Updating implicit path regardless!", n.FriendlyName())
```

Comment:
Can't comment on the right line: I think we should now also update the comment for this function, since now return a map of package names to implicit nodes, regardless if they were resolved byt a cached package or not.

Reply:
Actually, the name of the function itself might be a bit confusing.